.gitignore

```diff
@@ -1,3 +1,3 @@
-vendor
-composer.lock
```

Comment:
I'll let @gianarb decide on this one. Personally I'd be in favour of not committing the lock file and running tests with latest dependencies (as is the default in `composer install` without a lock file) and once with the `--prefer-lowest` flag set to install the lowest dependencies. This has multiple advantages: it makes sure the lower package constraints are still correct, and it would alleviate the problem Matthew indicated in the linked comment about developers not knowing if their PR broke due to their code changes or upstream package changes.